code/game/objects/items/movable_dispenser.dm

```diff
@@ -0,0 +1,66 @@
+/obj/item/movable_machinery
+	name = "undeployed ERROR"
+	desc = "something went wrong."
+	w_class = W_CLASS_LARGE
```

Comment:
should probably be huge given it's musttwohand and slows you down

Reply:
i copied it from gibtonite and yeah putting them into a BOH probably shouldn't work